HACK.md

```diff
@@ -45,133 +43,3 @@ make clean && make build
 
 * This project uses Golang 1.13+ and operator-sdk 1.15.1.
 * The controllers create/watch Tekton objects.
-
-## Unit tests
-
-### Counterfeiter
```

Comment:
Please provide a link to the new testing doc here.

Reply:
done